figures/models.py

```diff
@@ -391,6 +391,17 @@ def completed_ids_for_site(self, site, **_kwargs):
         qs = self.completed_for_site(site, **_kwargs)
         return qs.values('course_id', 'user_id').distinct()
 
+    def edly_completed_courses_for_site(self, site, **_kwargs):
```

Comment:
We already have many functions and we are not using the `edly` prefix. I think we should be consistent.